lib/track.ts

```diff
@@ -12,6 +12,12 @@ class MissingParamError extends Error {
   }
 }
 
+export enum Identifier {
+  ID = 'id',
+  EMAIL = 'email',
+  CIOID = 'cio_id',
+}
```

Comment:
Since this will be used in more places in the future, it would be best to move this to the `utils.ts` file. Additionally, our enum case naming conventions use `PascalCase`, similar to go.

```suggestion
export enum Identifier {
  Id = 'id',
  Email = 'email'
  CioId = 'cio_id'
}
```

Reply:
Sure, `types.ts` or `identifiers.ts` works. Probably the later, as more identifier functions and types may be added later